R/theme_behindbars.R

```diff
@@ -5,9 +5,9 @@
 #' @import ggplot2
 #' @export
 #'
-#' @param base_size base font size, given in pts 
+#' @param base_size base font size, given in pts
 #' @param base_family base font family
-#' 
+#'
 #' @examples
```

Comment:
Can we add an example of how to use the map here for the docs

Reply:
Yes, definitely! 

Separate but related: If we (as a team) like the U.S. state hex maps, I can also add a function to barstools that automates making them per our style guide.

Reply:
Love the idea. We should talk about it at our meeting today as well